charts/revere/templates/deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}-deployment
+  labels:
+    {{- include "revere.labels" . | nindent 4 }}
+spec:
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  revisionHistoryLimit: 0
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      deployment: {{ .Values.name }}-deployment
+  template:
+    metadata:
+      labels:
+        deployment: {{ .Values.name }}-deployment
+        {{- include "revere.labels" . | nindent 8 }}
+      annotations:
+        {{- /* Automatically restart deployments on config map change: */}}
+        checksum/{{ .Values.name }}-cm: {{ include (print $.Template.BasePath "/configMap.yaml") . | sha256sum }}
+        {{- range $key, $value := .Values.annotations }}
+        {{ $key }}: {{ $value }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ .Values.name }}-sa
+
+      {{- if .Values.runtimeClassName }}
+      runtimeClassName: {{ .Values.runtimeClassName }}
```

Comment:
Is this related to one of Dennis' suggestion. I don't recall seeing this field before?

Reply:
Yeah this is GKE sandbox capability. From an app's perspective, you just specify the runtime class ([link](https://cloud.google.com/kubernetes-engine/docs/how-to/sandbox-pods#sandboxed-application)), but it is more work to set up on a cluster (it is a flag you enable on a new node pool). I figure it would be a pretty minimal thing to expose here and we can discuss if adding a node pool is a route we want to go down.

Reply:
cool seems simple enough. Adding node pools is relatively easy via our cluster terraform module.